globals.go

```diff
@@ -505,26 +505,22 @@ type PatchPermission int
 type LogsPermission int
 
 const (
-	PermissionProjectSettings  = "project_settings"
-	PermissionProjectVariables = "project_variables"
-	PermissionTasks            = "project_tasks"
-	PermissionPatches          = "project_patches"
-	PermissionLogs             = "project_logs"
-
-	ProjectSettingsEdit  ProjectSettingsPermission  = 20
-	ProjectSettingsView  ProjectSettingsPermission  = 10
-	ProjectSettingsNone  ProjectSettingsPermission  = 0
-	ProjectVariablesEdit ProjectVariablesPermission = 20
-	ProjectVariablesView ProjectVariablesPermission = 10
-	ProjectVariablesNone ProjectVariablesPermission = 0
-	TasksAdmin           TasksPermission            = 30
-	TasksBasic           TasksPermission            = 20
-	TasksView            TasksPermission            = 10
-	TasksNone            TasksPermission            = 0
-	PatchSubmit          PatchPermission            = 10
-	PatchNone            PatchPermission            = 0
-	LogsView             LogsPermission             = 10
-	LogsNone             LogsPermission             = 0
+	PermissionProjectSettings = "project_settings"
+	PermissionTasks           = "project_tasks"
+	PermissionPatches         = "project_patches"
+	PermissionLogs            = "project_logs"
+
+	ProjectSettingsEdit ProjectSettingsPermission = 20
+	ProjectSettingsView ProjectSettingsPermission = 10
+	ProjectSettingsNone ProjectSettingsPermission = 0
```

Comment:
Are `ProjectVariables` permissions now part of `ProjectSettings` permissions?

Reply:
Yep, both will be controlled by that permission